### ISSUE TITLE
Weird install issues with latest 3 commits on master

### ISSUE THREAD

**Issue by Swizec** [webpack-contrib/worker-loader#104]
I have a strange issue and I think it can be fully resolved by an official release/update of `worker-loader`.

For my environment to work correctly I need this `publicPath` commit from last week: https://github.com/webpack-contrib/worker-loader/commit/96c6144d1aa32f599f41444fd24dc7d91028c1d7

I'm doing that by using this sort of package.json config:

```
 "worker-loader": "git://github.com/webpack-contrib/worker-loader#96c6144d1aa32f599f41444fd24dc7d91028c1d7"
```

But this commit from 3 days ago seems to be causing issues: https://github.com/webpack-contrib/worker-loader/commit/09705fb6faa8115685c0f185ef8d1e0ca39b0852

When I push to Heroku I now get a strange error (wasn't an issue last week)
```
remote: -----> Building dependencies
remote:        Installing node modules (package.json)
remote:        npm ERR! git checkout 2051a3ad661509e4c6a6952a6477299845317582: error: Your local changes to the following files would be overwritten by checkout:
remote:        npm ERR! git checkout 2051a3ad661509e4c6a6952a6477299845317582:     README.md
remote:        npm ERR! git checkout 2051a3ad661509e4c6a6952a6477299845317582: Please, commit your changes or stash them before you can switch branches.
remote:        npm ERR! git checkout 2051a3ad661509e4c6a6952a6477299845317582: Aborting
remote:        npm ERR! git checkout 2051a3ad661509e4c6a6952a6477299845317582:
remote:        npm ERR! Linux 3.13.0-133-generic
remote:        npm ERR! argv "/tmp/build_4b224c7f9a47f4800cb0b180fee54fb3/.heroku/node/bin/node" "/tmp/build_4b224c7f9a47f4800cb0b180fee54fb3/.heroku/node/bin/npm" "install" "--unsafe-perm" "--userconfig" "/tmp/build_4b224c7f9a47f4800cb0b180fee54fb3/.npmrc"
remote:        npm ERR! node v6.9.1
remote:        npm ERR! npm  v3.10.8
remote:        npm ERR! code 1
```

So just point to master or to that commit, right?

But then `worker-loader` installs as an empty package. These are all files in `node_modules/worker-loader`

```
$ ls node_modules/worker-loader 
CHANGELOG.md LICENSE      README.md    node_modules package.json
```

As you would expect, this leads to the issue that `worker-loader` doesn't exist when you try to use it:
```
Module not found: Error: Can't resolve 'worker-loader' in '/Users/Swizec/Documents/freelancing/math-crunch'
 @ ./app/assets/javascripts/Apps/TutorDashboardApp/Views/TutorShiftView.js 7:20-50
 @ ./app/assets/javascripts/Apps/TutorDashboardApp/SkeletonView.js
 @ ./app/assets/javascripts/Apps/TutorDashboardApp/index.js
```

To recap:

1. PR #31 works great locally
2. PR #31 doesn't deploy to Heroku because PR #81 causes checkout errors
3. PR #81 breaks `npm install` both locally and on Heroku

Is there an alternative to using PR #31 when trying to load everything except workers from cloudfront? If there is, then I could use the released version 1.0.0, which presumably works.

**Comment by michael-ciniawsky** [comment]
Install from github directly doesn't work anymore after #81, since the repo boilerplate applied there builds the files on `prepublish` and doesn't provide the `src` files. This needs a release

cc @d3viant0ne

**Comment by d3viant0ne** [comment]
~30 minutes

**Comment by d3viant0ne** [comment]
@Swizec - Available as `worker-loader@1.1.0`

**Comment by michael-ciniawsky** [comment]
Yup 😛  released in [v1.1.0](https://github.com/webpack-contrib/worker-loader/releases/tag/v1.1.0)

**Comment by Swizec** [comment]
Thanks @d3viant0ne and @michael-ciniawsky you're the best ❤️
